src/unstract/sdk/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -26,21 +26,17 @@
       "format": "password",
       "description": "API key obtained from the Unstract developer portal (https://unstract-api-resource.developer.azure-api.net)"
     },
-    "processing_mode": {
+    "mode": {
       "type": "string",
-      "title": "Processing Mode",
+      "title": "Mode",
       "enum": [
-        "text",
-        "ocr"
+        "native_text",
+        "low_cost",
+        "high_quality",
+        "form"
       ],
-      "default": "text",
-      "description": "Text mode tries to extract text from PDF and falls to OCR if the PDF is a scanned image PDF. This should be your default selection. Use OCR mode if you want to force OCR to extract text. This could be useful if you are dealing with malformed PDFs."
-    },
-    "force_text_processing": {
-      "type": "boolean",
-      "title": "Force Text Processing",
-      "default": false,
-      "description": "If checked, ensures that only text processing runs and there is no OCR involved. This differs from the default behaviour where we fall back to OCR processing in case of failures with text processing."
+      "default": "form",
+      "description": "Native text : Extracts text from PDF without OCR. This is very fast and cost effective. Use this mode if you are sure all your PDFs are native text pdfs (not scanned documents). Note that some scanned PDFs are \"searchable\" PDFs. Use the OCR modes for these PDFs as the quality of text in these documents are often poor. \n Low cost : Extracts text from scanned and native PDFs, images and office documents. This OCR mode cannot handle handwriting and low quality scanned pdfs and images. \n High quality : Extracts text from scanned and native PDFs, images and office documents. This OCR mode can handle handwriting and low quality scanned pdfs and images. \n Form:  Extracts text from scanned and native PDFs, images and office documents. This OCR mode can handle handwriting and low quality scanned pdfs and images. Can also extract information about checkboxes and radio button"
```

Comment:
NIT: @johnyrahul having a big text description is not readable. Either we should have a simple subtext in the UI (not sure if its possible with JSON schema) or we should have a clickable link that points to our docs. Please raise this as a concern with @shuveb, @arun-venkataswamy 
 and Tanushree to see how it can be improved

Reply:
UX part can be taken care as separate discussion.